pom.xml

```diff
@@ -439,11 +440,10 @@
     <!-- Make OSGi happy -->
     <osgi.snapshot.qualifier>${maven.build.timestamp}</osgi.snapshot.qualifier>
 
-    <version.io.netty>4.1.42.Final</version.io.netty>
     <!-- IMPORTANT: Don't use this dependency. The right dependency version is the one named as "version.io.netty".
          This is just needed by Elasticsearch Transport Plugin because it has a compatibility mode with netty 3 and
          it can't be removed -->
-    <version.io.netty.old>3.10.6.Final</version.io.netty.old>
+    <version.io.netty>3.10.6.Final</version.io.netty>
```

Comment:
Why are we basically reverting the change from an already merged PR? https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1095/files#diff-600376dffeb79835ede4a0b285078036R446

Reply:
good catch @MarianMacik this should not be reverted.

Is it a matter of rebasing or intentionally changed @cuijulian ?

Reply:
Sorry about this, yes it is from not rebasing on `master`. I will force push changes once the issues are resolved.